tests/store/model_registry/test_sqlalchemy_store.py

```diff
@@ -502,6 +506,10 @@ def test_search_registered_models(self):
         rms, _ = self._search_registered_models(f"name ILIKE '{prefix + 'cats'}%'")
         self.assertEqual(rms, [])
 
+        # confirm that ILIKE is not case-sensitive
+        rms, _ = self._search_registered_models(f"name ilike '%RM4a'")
```

Comment:
also add a negative test for case-insensitive operator : searching with filter `"name iLike '%RM17%'"` should yield empty result.

Reply:
Done!